ui/js/controllers/logviewer.js

```diff
@@ -269,7 +269,12 @@ logViewerApp.controller('LogviewerCtrl', [
 
                             if ($scope.step_data.all_errors.length == 0) {
                                 angular.element(document).ready(function () {
-                                    $scope.displayLog($scope.step_data.steps[0], 'initialLoad');
+                                    if (isNaN($scope.selectedBegin)) {
+                                        $scope.displayLog($scope.step_data.steps[0], 'initialLoad');
+                                    } else {
+                                        $scope.displayedStep = getStepFromLine($scope.selectedBegin);
+                                        moveScrollToLineNumber($scope.selectedBegin, $event);
```

Comment:
It looks like ``moveScrollToLineNumber`` always relies on the ``$scope.displayedStep`` value being set.  Since the value passed to ``getStepFromLine`` is always passed in to ``moveScrollToLineNumber``, we should just set ``$scope.displayedStep`` within that funciton and save the extra line each time.  Prevents code duplication.